pkg/volume/rbd/rbd_util.go

```diff
@@ -317,6 +318,34 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) error {
 	// mount it
 	if err = b.mounter.FormatAndMount(devicePath, globalPDPath, b.fsType, nil); err != nil {
 		err = fmt.Errorf("rbd: failed to mount rbd volume %s [%s] to %s, error %v", devicePath, b.fsType, globalPDPath, err)
+		// Currently, RBD plugin does `rbd lock/map` and `format/mount` in one
```

Comment:
@rootfs 
`DetachDisk` gets `devicePath` from mount point and need to report error if it fails to unmount the point. I cannot call it directly. I added a new subroutine `RBDUtil.detachDevice`.